packages/dashboard/src/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -157,10 +185,50 @@ function GoogleAnalyticsSettings({
       <div>
         <SettingHeading htmlFor="gaTrackingID" as="h3">
           {TEXT.SECTION_HEADING}
+          {usingLegacyAnalytics && isAutoAnalyticsMigrationEnabled && (
+            <StyledNotificationBubble notificationCount={1} isSmall />
+          )}
         </SettingHeading>
         <SettingSubheading size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
           {siteKitDisplayText}
         </SettingSubheading>
+        {usingLegacyAnalytics && isAutoAnalyticsMigrationEnabled && (
```

Comment:
This migration is only a WordPress thing right? This migration is only need for WordPress. Should we call this out and leave a todo for when break up the packages.

Reply:
Yes, it's WordPress-specific. Like a lot of other things in this component, e.g. the Site Kit information. No need for a todo in the code for this though, there will be an in-depth analysis of the dashboard as part of #2918 anyway.

Reply:
I think we should be adding todo to new functionality we know is WordPress. Why make more work for ourselves later to find it again?

Reply:
Such a todo doesn't need to be in the code, it can be noted externally in a doc.